### PR TITLE
Fix ify not preventing auto-pausing without premium

### PR DESCRIPTION
**Pull request by Ruthenic**

### CODE REVIEW

File: src/plugins/ify.ts [hunk 1]
```diff
@@ -11,11 +11,18 @@ export default definePlugin({
...
                 match: new RegExp(`${event}:function\\(.\\){.+?}(,|}\\))`),
                 replace: (_, ending) => `${event}:function(){}${ending}`,
-            })),
+            })), */
             ],
         },
+		{
+			find: "Playback auto paused",
+			replacement: {
+				match: /.{1,2}\.default\.track\(.{1,2}\..{1,3}\.SPOTIFY_AUTO_PAUSED\)/,
```
Comment (Vendicated) [inline src/plugins/ify.ts:23]: this just makes it not track it

Comment (Ruthenic) [reply src/plugins/ify.ts:23]: not in my testing

**Comment by Vendicated** [comment]
better approach: in same module as the old patches:
```js
match: /(?<=var.{0,10})isCurrentClientInVoiceChannel(),/,
replace: /false,
```

**Comment by Ruthenic** [comment]
re-review wen

File: src/plugins/ify.ts [hunk 2]
```diff
@@ -8,14 +8,17 @@ export default definePlugin({
...
                 replace: ".isPremium=true;",
-            }, ...["SPEAKING", "VOICE_STATE_UPDATES", "MEDIA_ENGINE_SET_DESKTOP_SOURCE"].map(event => ({
-                match: new RegExp(`${event}:function\\(.\\){.+?}(,|}\\))`),
-                replace: (_, ending) => `${event}:function(){}${ending}`,
-            })),
-            ],
+            }
         },
+		{
+			find: "Playback auto paused",
```
Comment (Vendicated) [inline src/plugins/ify.ts:17]: this is actually the same module so you can merge the replacements

Comment (Ruthenic) [reply src/plugins/ify.ts:17]: oh fair

**Comment by Ruthenic** [comment]
Make sure that commit makes sense i did it on my phone 💀

**Comment by Vendicated** [comment]
horror

**Comment by Ruthenic** [comment]
okay should work now :skull:

**Comment by Vendicated** [comment]
are u sure

**Comment by Ruthenic** [comment]
i literally compiled it and then tested it

**Pull request merged**